h/views/activity.py

```diff
@@ -75,14 +75,12 @@ def user_link(userid):
                                           username=username)
 
         return {
-            'aggregations': result.aggregations,
+            'search_results': results,
             'groups_suggestions': groups_suggestions,
-            'page': paginate(self.request, result.total, page_size=page_size),
+            'page': paginate(self.request, results.total, page_size=page_size),
             'pretty_link': pretty_link,
             'q': self.request.params.get('q', ''),
             'tag_link': tag_link,
-            'timeframes': result.timeframes,
-            'total': result.total,
```

Comment:
I can't seem to add comments to lines that are not in the initial set of the diff.

Line 408 in the search template still uses the old `total`, that needs to change to `search_results.total` as well